R/data_maggie.R

```diff
@@ -0,0 +1,464 @@
+#'@source Mediacloud \url{https://mediacloud.org/}
+#'
+#'Mediacloud Trump Data:
+#'@format A data frame with 51 rows representing dates and 7 variables:
+#'\describe{
+#'  \item{date}{Date}
+#'  \item{puerto_rico}{The number of headlines that mention Puerto Rico on the given date}
+#'  \item{puerto_rico_and_trump}{The number of headlines that mention Puerto Rico and either President or Trump}
+#'  \item{florida}{The number of headlines that mention Florida}
+#'  \item{florida_and_trump}{The number of headlines that mention Florida and either President or Trump}
+#'  \item{texas}{The number of headlines that mention Texas}
+#'  \item{texas_and_trump}{The number of headlines that mention Texas and either President or Trump}
+#'}
+#'@source Mediacloud \url{https://mediacloud.org/}
+#'
+#'TV Hurricanes Data:
+#'@format A data frame with 37 rows representing dates and 5 variables:
+#'\describe{
+#'  \item{date}{Date}
+#'  \item{harvey}{The percent of sentences in TV news that mention Hurricane Harvey on the given date}
+#'  \item{irma}{The percent of sentences in TV news that mention Hurricane Irma}
+#'  \item{maria}{The percent of sentences in TV news that mention Hurricane Maria}
+#'  \item{jose}{The percent of sentences in TV news that mention Hurricane Irma}
+#'}
+#'@source Internet TV News Archive \url{https://archive.org/details/tv} and 
+#'Television Explorer \url{https://television.gdeltproject.org/cgi-bin/iatv_ftxtsearch/iatv_ftxtsearch}
+#'
+#'TV Hurricanes by Network Data:
+#'@format A data frame with 84 rows representing dates and 6 variables:
+#'\describe{
+#'  \item{date}{Date}
+#'  \item{query}{The hurricane in question}
+#'  \item{bbc_news}{The percent of sentences on the BBC News TV channel on the given date that mention the hurricane in question}
+#'  \item{cnn}{The percent of sentences on CNN News that mention the hurricane in question}
+#'  \item{fox_news}{The percent of sentences on Fox News that mention the hurricane in question}
+#'  \item{msnbc}{The percent of sentences on MSNBC News that mention the hurricane in question}
+#'  }
+#'@source Internet TV News Archive \url{https://archive.org/details/tv} and 
+#'Television Explorer \url{https://television.gdeltproject.org/cgi-bin/iatv_ftxtsearch/iatv_ftxtsearch}
+#'
+#'TV States Data:
+#'@format A data frame with 52 rows representing dates and 4 variables:
+#'\describe{
+#'  \item{date}{Date}
+#'  \item{florida}{The percent of sentences in TV News on the given day that mention Florida}
+#'  \item{texas}{The percent of sentences in TV News on the given day that mention Texas}
+#'  \item{puerto_rico}{The percent of sentences in TV News on the given day that mention Puerto Rico}
+#'}
+#'@source Internet TV News Archive \url{https://archive.org/details/tv} and 
+#'Television Explorer \url{https://television.gdeltproject.org/cgi-bin/iatv_ftxtsearch/iatv_ftxtsearch}
+
+
+#TODO: Riddler dataset was preloaded in data-raw and is not linked in the article so I'm not sure what to source
```

Comment:
Let's drop this dataset entirely and mark it as such in the [master spreadsheet](https://docs.google.com/spreadsheets/d/1IMWAHNPIDzplafWW6AGnGyHmB1BMjohEw_V5HmT70Gs/edit). Don't forget to add your datasets there!